moon-store-mac-os/moon-store-mac-os/Services/BaseNetworkService.swift

```diff
@@ -38,7 +40,7 @@ class BaseNetworkService {
         components.path = path
         
         guard let url = components.url else {
-            throw MSError.badUrl
+            throw errorHelper.handle(error: .badUrl)
```

Comment:
Helper name no need manage instances. so when you use helper name in some class the most of the time that class use static func or is a base for other class because doesnt have other important functionality